### ISSUE TITLE
Is {EMAILICON} deprecated?

### ISSUE THREAD

**Issue by MikeyGMT** [e107inc/e107#2279]
Looking at email.php for lans rework, but looks like {EMAILICON} is no longer used on templates. So is email.php deprecated?

**Comment by rica-carv** [comment]
It might be, since now the email icon & button is on the [social plugin](https://github.com/e107inc/e107/blob/e05425fbb2b315a50d4e15dabe13c886b2803c93/e107_plugins/social/e_shortcode.php#L27)...
![untitled 1](https://cloud.githubusercontent.com/assets/17130530/21956942/58947bb0-da83-11e6-8ead-8e156924aff9.jpg)

**Comment by MikeyGMT** [comment]
Ah OK - have yet to play with social plugin. I'll leave the lans alone for emailicon.

**Comment by rica-carv** [comment]
But for some reason, i can't figure out why the send email button has anything to do with the social plugin, if there's already a core one....
Why no use it inside the social plugin?

Just my thoughts....

**Comment by MikeyGMT** [comment]
I'm looking to comment all unused LANs for #6 to reduce the number of LANs for translators. Hence, if they're not used, then let's disable them.

**Comment by Moc** [comment]
> But for some reason, i can't figure out why the send email button has anything to do with the social plugin, if there's already a core one....
Why no use it inside the social plugin?

Probably because it is a leftover from v1

**Comment by rica-carv** [comment]
@Moc ? 

> Probably because it is a leftover from v1

What's leftover from V1? The Email button shortcode, or the inclusion of it inside the social plugin (which i can't yet understand why it's there).

For instance, what if i want to use that button isolated fom all the social buttons?

**Comment by Moc** [comment]
As far as I understand it: {EMAILICON} was present in v1, now in v2 the social plugin combines a lot of these sharing options (social media but also email). There are thus two options to share via email: the emailicon and the social plugin. 

I may be incorrect though, I don't use either.

**Comment by rica-carv** [comment]
@Moc I also understand that, but in my v1.x site i had the regular legacy email button, wich allowed me also to put it wherever i want it. If now the legacy button is scheduled to go, in favor of the social plugin button row, there's a big issue in that ideia....
How can the social plugin render only the email button?

**Comment by Moc** [comment]
@CaMer0n What do you think?

**Comment by CaMer0n** [comment]
@Moc No, it's not deprecated.. as you can see here in the news shortcodes: 

	/**
	 * {EMAILICON: class=x}
	 */
	function sc_emailicon($parm=array())
	{
		$pref = e107::getPref();
		if (!check_class(varset($pref['email_item_class'],e_UC_MEMBER)))
		{
			return '';
		}
		require_once(e_HANDLER.'emailprint_class.php');
		return emailprint::render_emailprint('news', $this->news_item['news_id'], 1, $parm);
	}

It would have `@deprecated`  in the docs above the function/shortcode if it were.
